src/community/resources/contribution-day-organizer-handbook.md

```diff
@@ -0,0 +1,162 @@
+| 11AM-1PM  | Development Time                                                                  | Get hacking!                                                                                                                                                                                                                                                                                                                                                                     |
+| 1PM-2PM   | Lunch                                                                             | nom nom                                                                                                                                                                                                                                                                                                                                                                          |
+| 2PM-4PM   | Development Time                                                                  | Get hacking!                                                                                                                                                                                                                                                                                                                                                                     |
+| 4PM-5PM   | Demonstrations and Presentations                                                  | It has worked well in the past to finish the event with a demonstration of the work produced during the day. This way the group sees the larger picture of the event and not just the task(s) they have taken part in directly. The final portion of this time slot can be used to celebrate top achievements and hand out any prizes, if any.                                   |
+| 5 mins    | Thank you and Wrap Up                                                             | A final note from the main organizer to wrap up the event: thank everyone for attending and remind everyone to keep an eye on their inbox for a [feedback survey](#feedback).                                                                                                                                                                                                                 |
+
+### Tracking Contributions
+The Magento Community Engineering team would love it if you are able to track activity and contributions that arise out of your event! This helps us highlight the benefits for the community, from the community, to the Magento ecosystem and provides us with evidence for further supporting such community events in the future. We are particularly interested in understanding how a contributor's activity evolves before, during and after a Contribution Day event, so information we are most interested in is:
+
+*  Number of participants
+
+*  GitHub.com usernames of contributors
```

Comment:
Attendees MUST agree to this sharing of information

Reply:
Agree, it is probably better to not have such information, the list of PR is probably enough for us.

Reply:
I believe GitHub usernames are often required to provide additional access that may be needed during the event

Reply:
The topic on this one is related to `Tracking Contributions`, as currently, we have Bitergia to analyze the contributions I'm not sure if need that information. At the `Registration Page` it is still recommended for the organizer to collect GitHub username.